webpack/subscriptions_extension_index.js

```diff
@@ -1,8 +1,17 @@
-import { subscriptionCountListener } from './ForemanInventoryUpload/SubscriptionsPageExtension/SubscriptionsPageExtensionActions';
+import React from 'react';
+import { addGlobalFill } from 'foremanReact/components/common/Fill/GlobalFill';
+import { subscriptionCountListener, InventoryAutoUploadSwitcher } from './ForemanInventoryUpload/SubscriptionsPageExtension';
 
 if (window.location.pathname === '/subscriptions') {
```

Comment:
I suppose we can remove this, if we integrate the switch properly into the page.